src/common/components/img-gallery/index.jsx

```diff
@@ -1,36 +1,66 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
 
-import style from 'common/components/scss/ImgGallery.module.scss'
+import { requestImagesGallery } from 'redux/actions'
+import { array, bool, func, string, number } from 'prop-types'
+
+import { Preloader, Search, Pagination, SliderYears } from 'ui-kit'
 
 import noImage from 'assets/images/no-image.png'
+import style from 'common/components/scss/ImgGallery.module.scss'
 
-const ImageGallery = () => (
-  <div className={style.container}>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
-    </div>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
-    </div>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
-    </div>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
-    </div>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
-    </div>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
-    </div>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
-    </div>
-    <div className={style.container__content}>
-      <img src={noImage} alt="" />
+const ImagesGallery = ({ isLoading, images, onImagesGallery, searchText, page, yearStart, yearEnd }) => {
+  useEffect(() => {
+    onImagesGallery()
+  }, [onImagesGallery, searchText, page, yearStart, yearEnd])
+
+  const image = images.flatMap(img => img.links)
```

Comment:
you dont need to do it EVERY TIME inside the component. Make a function outside the component that will give you the same result